lib/std/build.zig

```diff
@@ -2574,6 +2580,11 @@ pub const LibExeObjStep = struct {
         }
 
         if (self.target.isDarwin()) {
```

Comment:
The whole patch looks good except I don't think we need the `isDarwin()` here. Other OS's have the concept of rpath too.

Reply:
Excellent point, thanks Andrew! I actually honestly thought the concept of `rpath`s is darwin specific :-)

Reply:
Done in 79d460a